src/main/java/seedu/address/logic/commands/general/GeneralCommand.java

```diff
@@ -0,0 +1,9 @@
+package seedu.address.logic.commands.general;
+
+import seedu.address.logic.commands.Command;
+
+/**
+ * Represents all commands in the general package which will not cause commit changes to model history.
+ */
+public abstract class GeneralCommand extends Command {
```

Comment:
It feels weird to have an empty abstract class. How about including an abstract `execute` method since all subclasses have them?

Reply:
`execute` is already an abstract method from `Command`